webpack.config.js

```diff
@@ -24,7 +24,11 @@ module.exports = {
       exclude: /node_modules/,
     }, {
       test: /\.scss$/,
-      loaders: ['style-loader', 'css-loader', 'sass-loader'],
+      loaders: [
```

Comment:
use webpack2 syntax if possible for loaders 

https://webpack.js.org/concepts/loaders/